src/Paystack.php

```diff
@@ -278,4 +278,264 @@ private function getData()
         return $this->getResponse()['data'];
     }
 
-}
\ No newline at end of file
+    /**
+     * Create a plan
+     */
+    public function createPlan(){
+
+        $data = [
+            "name" => request()->name,
+            "description" => request()->desc,
+            "amount" => intval(request()->amount),
+            "interval" => request()->interval,
+            "send_invoices" => request()->send_invoices,
+            "send_sms" => request()->send_sms,
+            "currency" => request()->currency,
+        ];
+
+        $this->setRequestOptions();
+
+        $this->response = $this->setGetResponse("/plan", $data);
```

Comment:
Meanwhile, you are creating a plan, isn't that a post request?..How come you are using a get request here?